UnityProject/Assets/Scripts/Health/Living/DNABloodTypeGen.cs

```diff
@@ -0,0 +1,60 @@
+using Random = UnityEngine.Random;
+
+[System.Serializable]
+public class DNAandBloodType
+{
+	public string DNAString;
+	public BloodTypes BloodType;
+	
+	private int BloodTypeGenerator;
+	
+	public void BloodTypeandDNA()
```

Comment:
This is no longer a constructor. If you don't want to use the constructor then you have to at least call this method to do the generation once you've created the class to be stored in the DNA field.

Reply:
The C# documentation explains constructors pretty well: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constructors